src/ResourceManagement/Insights/Microsoft.Azure.Insights/Generated/Insights/Models/MetricValue.cs

```diff
@@ -46,8 +46,8 @@ public MetricValue(System.DateTime timestamp, double? average = default(double?)
         /// <summary>
         /// Gets or sets the timestamp for the metric value in ISO 8601 format.
         /// </summary>
-        [Newtonsoft.Json.JsonProperty(PropertyName = "timestamp")]
-        public System.DateTime Timestamp { get; set; }
+        [Newtonsoft.Json.JsonProperty(PropertyName = "timeStamp")]
```

Comment:
This only fix a bug in Python and Node.js. The serialization/deserialization could fail in those cases and I did not detect the issue in my Node.js test cases. I am not writing the Python SDK, the I was told that my examples in the documentation was causing the Python SDK to fail because I was using "timestamp" instead of "timeStamp".
This has no effect in C# apparently because Newtonsoft.json is taking care of the different casing: it receives "timeStamp" through the cable, but it maps it to "Timestamp" in the model.
BTW: the comment about deprecation and replacement is valid here too.